src/webhook.rs

```diff
@@ -635,113 +663,135 @@ async fn handle_comment(
 				number,
 			))))?;
 		}
-	} else if body.to_lowercase().trim() == BURNIN_REQUEST.to_lowercase().trim()
+	} else if body
+		.to_lowercase()
+		.trim()
+		.starts_with(BURNIN_REQUEST.to_lowercase().trim())
```

Comment:
I don't know if the commands are known to be ASCII, but if that's the case then we might want to simplify that by calling `str::eq_ignore_ascii_case` on the `trim`med strings